content/en/docs/guides/build-deploy-software-via-helm-charts-with-ocm-101.md

```diff
@@ -0,0 +1,629 @@
+---
+title: "Build & Deploy Infrastructure via Helm Charts with OCM" 
+description: ""
+lead: ""
+date: 2024-03-19T10:36:48+01:00
+lastmod: 2024-03-19T10:36:48+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: ""
+    identifier: "build-deploy-software-via-helm-charts-with-ocm"
+weight: 101
+toc: true
+---
+
+## Introduction
+
+Let's illustrate a very simple "Hello World" example application and show how to leverage OCM to build and deploy a Helm Chart and OCI Image on a local `kind` k8s cluster.
+The topic of `ocm` [`localization`](https://ocm.software/docs/guides/complex-component-structure-deployment/#localization) and [`configuration`](https://ocm.software/docs/guides/complex-component-structure-deployment/#configuration) is NOT part of this very simple example and is part of other in deepth guides.
+
+As base we use the `podinfo` application.
+All files can be found [here](https://github.com/stb1337/ocm-hello-world-v1).
+
+At the end of the tutorial the application will consist of the following components:
+
+* `podinfo` as our business application
```

Comment:
since it is just one component I propose to skip the list and the following sentence in line 29 in favour of a single, combined sentence describing the result, an OCM component and its two versions, containing the business application podinfo. I'm not sure if mentioning the six OCI images makes sense here.

Reply:
I did a change in your main branch to this section. Please check if this make sense.